cluster/gce/upgrade-aliases.sh

```diff
@@ -124,9 +130,12 @@ function add-k8s-subnet-secondary-ranges() {
 function delete-k8s-node-routes() {
   local -a routes
   local -r batch=200
-  routes=( $(gcloud compute routes list \
-    --project=${PROJECT} --filter='description=k8s-node-route' \
-    --format='value(name)') )
+  routes=()
+  while IFS=$'\n' read -r route; do
+    routes+=( "${route}" )
+  done < <(gcloud compute routes list \
+    --project="${PROJECT}" --filter='description=k8s-node-route' \
+    --format='value(name)')
```

Comment:
change reason:
```
In ./cluster/gce/upgrade-aliases.sh line 133:
  routes=( $(gcloud compute routes list \
           ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```